src/commons/Constants.ts

```diff
@@ -131,15 +131,14 @@ const constants = {
       (screenHeight >= 812 || screenWidth >= 812);
   },
   /* Orientation */
+  dimensionsEventListener: undefined,
```

Comment:
Is `dimensionsEventListener` being used anywhere?

Reply:
Yes, It is used for fixing the layout when orientation change in Carousel, ColorPalette, GridView, Slider, Wizard, and useOrientation hook.